packages/terraform-modules-public/aws/testnet/modules/startup-scripts/run-attestation-service.sh

```diff
@@ -24,17 +24,37 @@ else
   DOCKER_LOGGING_PARAMS="--log-driver=awslogs --log-opt awslogs-group=$ATTESTATION_NODE_CLOUDWATCH_LOG_GROUP_NAME --log-opt awslogs-stream=$ATTESTATION_NODE_CLOUDWATCH_LOG_STREAM_NAME"
 fi
 
-docker run -d --name celo-attestations $DOCKER_LOGGING_PARAMS --restart always -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --nousb --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock 0x$CELO_ATTESTATION_SIGNER_ADDRESS --password /root/.celo/.password --bootnodes $BOOTNODE_ENODES --allow-insecure-unlock
-
+docker run -d --name celo-attestations $DOCKER_LOGGING_PARAMS --restart always -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --nousb --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin --unlock $CELO_ATTESTATION_SIGNER_ADDRESS --password /root/.celo/.password --bootnodes $BOOTNODE_ENODES --allow-insecure-unlock
 
 export CELO_IMAGE_ATTESTATION=${celo_image_attestation}
-export CELO_PROVIDER=http://localhost:8545
+export CONFIG_FILE_PATH=.attestationconfig
+
+echo 'DATABASE_URL=${database_url}' >> $CONFIG_FILE_PATH
```

Comment:
Does this startup script get re-run upon VM restart? Wondering if a restart and the usage of `>>` could result in duplicates for all these env vars

Reply:
It only runs once on VM creation.